### ISSUE TITLE
$.fn.dataTable.util.throttle is not overridable

### ISSUE THREAD

**Issue by fabiospampinato** [DataTables/DataTables#810]
It's currently not possible to override the function in question, since DT'll use another reference to it anyway. Why not calling this method instead?

For instance I might want to replace your implementation with lodash's, to have more options available while still retaining backward compatibility.

I may also want to replace it with `lodash.debounce` since it's not currenlty possible to easily use debouncing instead of throttling (#809).

**Comment by DataTables** [comment]
Good point - it should be! This would be a sort term solution to using debounce rather than throttle. I'll try to get this in for the 1.10.12 release and will post back here when done.

**Comment by DataTables** [comment]
[Fix committed here](https://github.com/DataTables/DataTablesSrc/commit/6646b5f07d2bc16733788fdda9cfda91cdb33dc9).

**Comment by DataTables** [comment]
Bother. I'll come back to it when I have a bit of time to do so. Thanks for pointing that out.

**Comment by PhilippMeissner** [comment]
Hi there. I updated DataTables to 1.10.16 and would like to swap out the method in question for the `debounce` method provided by Lodash. How would I do so? I can't seem to find any documentation about this.

**Comment by DataTables** [comment]
In theory all you should need to do is `$.fn.DataTable.util.throttle = ...` where `...` is your new throttle function. It it sounds like that isn't working correctly at the moment and I need to find some time to address it.
